ingeniamotion/wizard_tests/base_test.py

```diff
@@ -51,12 +51,12 @@ def __init__(self) -> None:
 
     def save_backup_registers(self) -> None:
         self.backup_registers[self.axis] = {}
-        for uid in self.backup_registers_names:
-            try:
+        try:
+            for uid in self.backup_registers_names:
                 value = self.mc.communication.get_register(uid, servo=self.servo, axis=self.axis)
                 self.backup_registers[self.axis][uid] = value
-            except IMRegisterNotExist as e:
-                self.logger.warning(e, axis=self.axis)
+        except IMRegisterNotExist as e:
```

Comment:
This is changing the behaviour of the wizard!
When a register is not found it will tell you that the register is ignored, but in fact all of the remaining ones are

Reply:
Done